web-ng/src/app/components/layer-dialog/layer-dialog.component.ts

```diff
@@ -65,6 +66,24 @@ export class LayerDialogComponent implements OnDestroy {
     );
   }
 
+  getFieldType() {
+    return {
+      icon: 'short_text',
+      label: 'Text',
+    };
+  }
+
+  initQuestion() {
+    return this.formBuilder.group({
+      en: [''],
+    });
+  }
+
+  addQuestion() {
+    const control = this.layerForm.controls['questions'] as FormArray;
```

Comment:
Where does the field `controls` come from? We should be using the existing `Form` model object, based on this data model:
https://github.com/google/ground-platform/wiki/Cloud-Firestore-Representation

Reply:
Ah sorry, misunderstood that this is an Angular form, not a Ground form :) Please disregard.